ext/fs/std_fs.rs

```diff
@@ -821,6 +821,43 @@ fn stat_extra(
     Ok(info.dwVolumeSerialNumber as u64)
   }
 
+  unsafe fn get_change_time(
+    handle: winapi::shared::ntdef::HANDLE,
+  ) -> std::io::Result<u64> {
+    use winapi::shared::minwindef::FALSE;
+    use winapi::um::fileapi::FILE_BASIC_INFO;
+    use winapi::um::minwinbase::FileBasicInfo;
+    use winapi::um::winbase::GetFileInformationByHandleEx;
+
+    let file_info = {
+      let mut file_info = std::mem::MaybeUninit::<FILE_BASIC_INFO>::zeroed();
+      if GetFileInformationByHandleEx(
+        handle,
+        FileBasicInfo,
+        &mut file_info as *mut _ as *mut _,
+        std::mem::size_of::<FILE_BASIC_INFO>() as u32,
+      ) == FALSE
+      {
+        return Err(std::io::Error::last_os_error());
+      }
+
+      file_info.assume_init()
+    };
+
+    let change_time = file_info.ChangeTime.QuadPart();
+    let unix_time_msec = windows_time_to_unix_time_msec(change_time);
+
+    Ok(unix_time_msec as u64)
+  }
+
+  const WINDOWS_TICK: i64 = 10_000; // 100-nanosecond intervals in a millisecond
```

Comment:
I don't think that this is a good place for consts, but had no idea where to put them, so I will be happy for some suggestions